test/functional/Java8andUp/src/org/openj9/test/attachAPI/AttachAPISanity.java

```diff
@@ -45,7 +44,7 @@
 	@Test(groups = { "level.sanity" })
 	public void testAttachToSelf() {
 		logger.debug("testAttachToSelf");
-		String myVmid = com.ibm.tools.attach.target.AttachHandler.getVmId();
+		String myVmid = getVmId();
```

Comment:
Suggest you use `TargetManager.getVmId` in preference to a static import.

Reply:
Even better, move the code into `setUp()`.